biolabalo/SendIT
# setup babel

#### What does this pull request do? 
- Transpile higher order  javascript  to es5 

#### Description of task completed?
- Install  babel-core, babel-register, babel-cli and babel-presets-env 
- Added and configured .babelrc file 

#### How should this be manually tested?
- Clone the repo , cd into sendIT/sever 
- write es6 code in app.js file
- run  node --require babel-register app.js
- You should see "Server started at localhost:5000"  in  the terminal

#### what are the relevant pivotal tracker stories?
- [#161716966]
